go/tests.yaml

```diff
@@ -60,6 +60,7 @@ files:
       go_test_cmd: ./test
       go_generated_dirs: [config, docs]
       go_generate_with_schematyper: true
+      go_versions: [1.19.x, 1.20.x, 1.21.x]
```

Comment:
I think instead we should probably be editing the defaults in `config.yaml` so it applies to all golang projects in this repo?

Reply:
Also, looking more closely, we're build with golang v1.20 in Fedora, CentOS Stream, and RHEL (at least for the streams we care about). So I think we can drop v1.19 as well (by the golang support policy, v1.19 is now no longer supported).

Reply:
Ah; ok I can do that though I was a little hesitant as it would have implications on other go-streams
I can update those too as well though.